### PR TITLE
Add the ability to truncate number if needed

### PR DESCRIPTION
**Pull request by innovimax**
!!!! WAS not tested before sending !!!!

### CODE REVIEW

**Comment by Pascalco** [comment]
What is an usecase where users want to truncate floating numbers?

File: index.php [hunk 1]
```diff
@@ -137,6 +137,7 @@
                             <option>.</option>
                             <option>,</option>
                         </select>
+                        <input type="checkbox" name="truncate" value="1">truncate the value on decimal mark</div>
```
Comment (matejsuchanek) [inline index.php]: where do you open the <div>?

**Comment by innovimax** [comment]
The use case is that for example for length of river in Wikidata it is expected to have a integer size in kilometer while in Wikipedia French they use floating point number

**Comment by Pascalco** [comment]
Length of river don't have to be integers on Wikidata. Same as on Wikipedia they can be floating point numbers.
Only this tool doesn't yet support to import floating point numbers.

**Comment by Pascalco** [comment]
floating point numberes can now be parsed. So I don't see any further usecase of this commit.

**Pull request closed without merging**